aashish/base.tf

```diff
@@ -0,0 +1,104 @@
+variable "region" {
+  type = string
+}
+
+variable "cidr_block" {
+  type    = string
+  default = "10.10.0.0/16"
```

Comment:
- CIDR blocks like `10.10.0.0/16` is a very extensive range, prefer to use like /24 or /26 if you do not have large number of IPs to assign in a VPC. AWS minimum range is /28. 
Why it matters?
If you have large IP ranges then it can conflict with other VPC IP ranges, you will have issues in VPC Peering.

Shy away from passing default values for VPC IP. Let your code ask explicitly for VPC IP range.

Reply:
Understood